src/css/main.css

```
@@ -153,3 +153,17 @@ nav.menu .menu__list-item--collapsed .menu__link--sublist:after,
     color: #98a3ff;
   }
 }
+
+/* Custom styling for code blocks */
+.theme-code-block {
+  overflow-x: auto; 
+  max-width: 100%; 
+  width: 100%; 
+  box-sizing: border-box;
+}
+/* Custom styling for code blocks content */
+.theme-code-block pre {
+  white-space: pre-wrap; 
+  word-wrap: break-word; 
```

Regarding https://github.com/seqeralabs/docs/pull/362/files#r1891721828 I think it's actually the `white-space` that needs care here. I guess that `word-wrap` will be ignored if using `white-space: pre` so can probably be removed.

```suggestion
  white-space: pre; 
```